dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -336,34 +336,6 @@
                                         <Setter Target="ContentPresenter.Foreground" Value="{ThemeResource NavigationViewButtonForegroundDisabled}" />
                                     </VisualState.Setters>
                                 </VisualState>
-                                <VisualState x:Name="Checked">
```

Comment:
Are these changes intentional ?

Reply:
Yes, this is the style for the hamburger button which is a button, not a toggle button.  These states have never been used.